core/float/round_spec.rb

```diff
@@ -65,11 +65,23 @@
     0.42.round(2.0**30).should == 0.42
   end
 
+  ruby_version_is "2.4" do
+    it "returns big values rounded to nearest even" do
+      +2.5e20.round(-20).should   eql( +2 * 10 ** 20  )
+      -2.5e20.round(-20).should   eql( -2 * 10 ** 20  )
+    end
+  end
+
+  ruby_version_is ""..."2.4" do
+    it "returns big values rounded to nearest" do
+      +2.5e20.round(-20).should   eql( +3 * 10 ** 20  )
+      -2.5e20.round(-20).should   eql( -3 * 10 ** 20  )
+    end
+  end
```

Comment:
Could you swap these two ìt` so the 2.4 behavior comes after the < 2.4 behavior?